### ISSUE TITLE
The videos in ASL SB don't always display ('Not Found')

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/NGT-signbank#392]
* Related to #360 ?
* Example: https://aslsignbank.haskins.yale.edu//dictionary/gloss/2639
* Didn't yet check if the file uploaded correctly, easier for you to do now that your fluent in yale-vpn @Woseseltops :-)

**Comment by ocrasborn** [comment]
Did do this test: uploaded a fresh video for https://aslsignbank.haskins.yale.edu/dictionary/gloss/2646?edit
Same error.

**Comment by Woseseltops** [comment]
Turns out new uploaded videos didn't get the correct permissions in all cases; a file with incorrect permissions cannot be shown on the website. If I'm not mistaken, new videos always get the correct permissions now.

**Comment by ocrasborn** [comment]
Nope, unfortunately not @Woseseltops . Just re-uploaded the same video to my test sign, same error. https://aslsignbank.haskins.yale.edu/dictionary/gloss/2646

**Comment by Woseseltops** [comment]
Thanks for retesting, @ocrasborn . I thought I tested correctly, but apparently not - I could reproduce your problem. 

I have two hypotheses about what's going wrong here: either the permission system works slightly different for RedHat (vs Ubuntu), or it's something related to uploaded files bringing along permissions from their source system. I tried for a long time to fix this in the OS, where I believe it should be fixed, but this was taking too much time. I've now added a single line of Python that fixes the permissions of all uploaded videos.
